filters/log/log.go

```diff
@@ -27,6 +29,12 @@ const (
 	// reject reason information into the state bag to pass the
 	// information to the auditLog filter.
 	AuthRejectReasonKey = "auth-reject-reason"
+	// UnverifiedAuditLogName is th filtername seend by the user
+	UnverifiedAuditLogName = "unverifiedAuditLog"
```

Comment:
not sure about this name. Maybe "subClaim" or "oauth2SubClaim"?

Reply:
No, because it's not getting a claim. It's also not a verified audit log, just a shorthand base64 decode of the middle part of a jwt token to make it possible to find the correlate logs. It's not meant for security auditlogging.
These are the reasons why I don't want to write down miss leading information to users.